2023.01.13 14:58:01 INFO  tracing is disabled for protocol LSP, to enable tracing of incoming and outgoing JSON messages create an empty file at /sgoinfre/scristia/codam/cub3d/.metals/lsp.trace.json or /Users/scristia/Library/Caches/org.scalameta.metals/lsp.trace.json
2023.01.13 14:58:02 INFO  logging to file /sgoinfre/scristia/codam/cub3d/.metals/metals.log
2023.01.13 14:58:02 INFO  Started: Metals version 0.11.10 in workspace '/sgoinfre/scristia/codam/cub3d' for client Visual Studio Code 1.74.3.
SLF4J: No SLF4J providers were found.
SLF4J: Defaulting to no-operation (NOP) logger implementation
SLF4J: See https://www.slf4j.org/codes.html#noProviders for further details.
SLF4J: Class path contains SLF4J bindings targeting slf4j-api versions 1.7.x or earlier.
SLF4J: Ignoring binding found at [jar:file:/Users/scristia/Library/Caches/Coursier/v1/https/repo1.maven.org/maven2/com/outr/scribe-slf4j_2.13/3.10.5/scribe-slf4j_2.13-3.10.5.jar!/org/slf4j/impl/StaticLoggerBinder.class]
SLF4J: See https://www.slf4j.org/codes.html#ignoredBindings for an explanation.
2023.01.13 14:58:04 WARN  resetting database: /sgoinfre/scristia/codam/cub3d/.metals/metals.h2.db
2023.01.13 14:58:04 ERROR org.flywaydb.core.internal.exception.FlywaySqlException: Unable to obtain connection from database (jdbc:h2:file:/sgoinfre/scristia/codam/cub3d/.metals/metals;AUTO_SERVER=TRUE) for user 'sa': IO Exception: "/sgoinfre/scristia/codam/cub3d/.metals/metals.mv.db" [90028-214]
----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------
SQL State  : 90028
Error Code : 90028
Message    : IO Exception: "/sgoinfre/scristia/codam/cub3d/.metals/metals.mv.db" [90028-214]

	at org.flywaydb.core.internal.jdbc.JdbcUtils.openConnection(JdbcUtils.java:60)
	at org.flywaydb.core.internal.jdbc.JdbcConnectionFactory.<init>(JdbcConnectionFactory.java:75)
	at org.flywaydb.core.FlywayExecutor.execute(FlywayExecutor.java:146)
	at org.flywaydb.core.Flyway.clean(Flyway.java:213)
	at scala.meta.internal.metals.Tables.migrateOrRestart(Tables.scala:156)
	at scala.meta.internal.metals.Tables.tryUrl(Tables.scala:144)
	at scala.meta.internal.metals.Tables.persistentConnection(Tables.scala:138)
	at scala.meta.internal.metals.Tables.tryAutoServer(Tables.scala:81)
	at scala.meta.internal.metals.Tables.connect(Tables.scala:48)
	at scala.meta.internal.metals.Tables.connection(Tables.scala:74)
	at scala.meta.internal.metals.Tables.$anonfun$fingerprints$1(Tables.scala:39)
	at scala.meta.internal.metals.Fingerprints.load(Fingerprints.scala:52)
	at scala.meta.internal.metals.MetalsLanguageServer.$anonfun$initialize$2(MetalsLanguageServer.scala:882)
	at scala.concurrent.Future$.$anonfun$apply$1(Future.scala:678)
	at scala.concurrent.impl.Promise$Transformation.run(Promise.scala:467)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
	at java.lang.Thread.run(Thread.java:833)
Caused by: org.h2.jdbc.JdbcSQLNonTransientException: IO Exception: "/sgoinfre/scristia/codam/cub3d/.metals/metals.mv.db" [90028-214]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:554)
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:477)
	at org.h2.message.DbException.get(DbException.java:212)
	at org.h2.mvstore.db.Store.convertMVStoreException(Store.java:171)
	at org.h2.mvstore.db.Store.<init>(Store.java:145)
	at org.h2.engine.Database.<init>(Database.java:324)
	at org.h2.engine.Engine.openSession(Engine.java:92)
	at org.h2.engine.Engine.openSession(Engine.java:222)
	at org.h2.engine.Engine.createSession(Engine.java:201)
	at org.h2.engine.SessionRemote.connectEmbeddedOrServer(SessionRemote.java:338)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:122)
	at org.h2.Driver.connect(Driver.java:59)
	at org.flywaydb.core.internal.jdbc.DriverDataSource.getConnectionFromDriver(DriverDataSource.java:258)
	at org.flywaydb.core.internal.jdbc.DriverDataSource.getConnection(DriverDataSource.java:223)
	at org.flywaydb.core.internal.jdbc.JdbcUtils.openConnection(JdbcUtils.java:48)
	at org.flywaydb.core.internal.jdbc.JdbcConnectionFactory.<init>(JdbcConnectionFactory.java:75)
	at org.flywaydb.core.FlywayExecutor.execute(FlywayExecutor.java:146)
	at org.flywaydb.core.Flyway.clean(Flyway.java:213)
	at scala.meta.internal.metals.Tables.migrateOrRestart(Tables.scala:156)
	at scala.meta.internal.metals.Tables.tryUrl(Tables.scala:144)
	at scala.meta.internal.metals.Tables.persistentConnection(Tables.scala:138)
	at scala.meta.internal.metals.Tables.tryAutoServer(Tables.scala:81)
	at scala.meta.internal.metals.Tables.connect(Tables.scala:48)
	at scala.meta.internal.metals.Tables.connection(Tables.scala:74)
	at scala.meta.internal.metals.Tables.$anonfun$fingerprints$1(Tables.scala:39)
	at scala.meta.internal.metals.Fingerprints.load(Fingerprints.scala:52)
	at scala.meta.internal.metals.MetalsLanguageServer.$anonfun$initialize$2(MetalsLanguageServer.scala:882)
	at scala.concurrent.Future$.$anonfun$apply$1(Future.scala:678)
	at scala.concurrent.impl.Promise$Transformation.run(Promise.scala:467)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
	at java.lang.Thread.run(Thread.java:833)
Caused by: org.h2.mvstore.MVStoreException: Could not open file /sgoinfre/scristia/codam/cub3d/.metals/metals.mv.db [2.1.214/1]
	at org.h2.mvstore.DataUtils.newMVStoreException(DataUtils.java:1004)
	at org.h2.mvstore.FileStore.open(FileStore.java:185)
	at org.h2.mvstore.FileStore.open(FileStore.java:128)
	at org.h2.mvstore.MVStore.<init>(MVStore.java:452)
	at org.h2.mvstore.MVStore$Builder.open(MVStore.java:4082)
	at org.h2.mvstore.db.Store.<init>(Store.java:136)
	at org.h2.engine.Database.<init>(Database.java:324)
	at org.h2.engine.Engine.openSession(Engine.java:92)
	at org.h2.engine.Engine.openSession(Engine.java:222)
	at org.h2.engine.Engine.createSession(Engine.java:201)
	at org.h2.engine.SessionRemote.connectEmbeddedOrServer(SessionRemote.java:338)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:122)
	at org.h2.Driver.connect(Driver.java:59)
	at org.flywaydb.core.internal.jdbc.DriverDataSource.getConnectionFromDriver(DriverDataSource.java:258)
	at org.flywaydb.core.internal.jdbc.DriverDataSource.getConnection(DriverDataSource.java:223)
	at org.flywaydb.core.internal.jdbc.JdbcUtils.openConnection(JdbcUtils.java:48)
	at org.flywaydb.core.internal.jdbc.JdbcConnectionFactory.<init>(JdbcConnectionFactory.java:75)
	at org.flywaydb.core.FlywayExecutor.execute(FlywayExecutor.java:146)
	at org.flywaydb.core.Flyway.clean(Flyway.java:213)
	at scala.meta.internal.metals.Tables.migrateOrRestart(Tables.scala:156)
	at scala.meta.internal.metals.Tables.tryUrl(Tables.scala:144)
	at scala.meta.internal.metals.Tables.persistentConnection(Tables.scala:138)
	at scala.meta.internal.metals.Tables.tryAutoServer(Tables.scala:81)
	at scala.meta.internal.metals.Tables.connect(Tables.scala:48)
	at scala.meta.internal.metals.Tables.connection(Tables.scala:74)
	at scala.meta.internal.metals.Tables.$anonfun$fingerprints$1(Tables.scala:39)
	at scala.meta.internal.metals.Fingerprints.load(Fingerprints.scala:52)
	at scala.meta.internal.metals.MetalsLanguageServer.$anonfun$initialize$2(MetalsLanguageServer.scala:882)
	at scala.concurrent.Future$.$anonfun$apply$1(Future.scala:678)
	at scala.concurrent.impl.Promise$Transformation.run(Promise.scala:467)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
	at java.lang.Thread.run(Thread.java:833)
Caused by: java.io.IOException: Operation not supported
	at sun.nio.ch.FileDispatcherImpl.lock0(Native Method)
	at sun.nio.ch.FileDispatcherImpl.lock(FileDispatcherImpl.java:96)
	at sun.nio.ch.FileChannelImpl.tryLock(FileChannelImpl.java:1304)
	at java.nio.channels.FileChannel.tryLock(FileChannel.java:1191)
	at org.h2.mvstore.FileStore.open(FileStore.java:169)
	at org.h2.mvstore.FileStore.open(FileStore.java:128)
	at org.h2.mvstore.MVStore.<init>(MVStore.java:452)
	at org.h2.mvstore.MVStore$Builder.open(MVStore.java:4082)
	at org.h2.mvstore.db.Store.<init>(Store.java:136)
	at org.h2.engine.Database.<init>(Database.java:324)
	at org.h2.engine.Engine.openSession(Engine.java:92)
	at org.h2.engine.Engine.openSession(Engine.java:222)
	at org.h2.engine.Engine.createSession(Engine.java:201)
	at org.h2.engine.SessionRemote.connectEmbeddedOrServer(SessionRemote.java:338)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:122)
	at org.h2.Driver.connect(Driver.java:59)
	at org.flywaydb.core.internal.jdbc.DriverDataSource.getConnectionFromDriver(DriverDataSource.java:258)
	at org.flywaydb.core.internal.jdbc.DriverDataSource.getConnection(DriverDataSource.java:223)
	at org.flywaydb.core.internal.jdbc.JdbcUtils.openConnection(JdbcUtils.java:48)
	at org.flywaydb.core.internal.jdbc.JdbcConnectionFactory.<init>(JdbcConnectionFactory.java:75)
	at org.flywaydb.core.FlywayExecutor.execute(FlywayExecutor.java:146)
	at org.flywaydb.core.Flyway.clean(Flyway.java:213)
	at scala.meta.internal.metals.Tables.migrateOrRestart(Tables.scala:156)
	at scala.meta.internal.metals.Tables.tryUrl(Tables.scala:144)
	at scala.meta.internal.metals.Tables.persistentConnection(Tables.scala:138)
	at scala.meta.internal.metals.Tables.tryAutoServer(Tables.scala:81)
	at scala.meta.internal.metals.Tables.connect(Tables.scala:48)
	at scala.meta.internal.metals.Tables.connection(Tables.scala:74)
	at scala.meta.internal.metals.Tables.$anonfun$fingerprints$1(Tables.scala:39)
	at scala.meta.internal.metals.Fingerprints.load(Fingerprints.scala:52)
	at scala.meta.internal.metals.MetalsLanguageServer.$anonfun$initialize$2(MetalsLanguageServer.scala:882)
	at scala.concurrent.Future$.$anonfun$apply$1(Future.scala:678)
	at scala.concurrent.impl.Promise$Transformation.run(Promise.scala:467)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
	at java.lang.Thread.run(Thread.java:833)

2023.01.13 14:58:04 WARN  resetting database: /sgoinfre/scristia/codam/cub3d/.metals/metals.h2.db
2023.01.13 14:58:04 ERROR unable to setup persistent H2 database with AUTO_SERVER=false, falling back to in-memory database. This means you may be redundantly asked to execute 'Import build', even if it's not needed. Also, code navigation will not work for existing files in the .metals/readonly/ directory. To fix this problem, make sure you only have one running Metals server in the directory '/sgoinfre/scristia/codam/cub3d'.org.flywaydb.core.internal.exception.FlywaySqlException: Unable to obtain connection from database (jdbc:h2:file:/sgoinfre/scristia/codam/cub3d/.metals/metals;AUTO_SERVER=TRUE) for user 'sa': IO Exception: "/sgoinfre/scristia/codam/cub3d/.metals/metals.mv.db" [90028-214]
----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------
SQL State  : 90028
Error Code : 90028
Message    : IO Exception: "/sgoinfre/scristia/codam/cub3d/.metals/metals.mv.db" [90028-214]

	at org.flywaydb.core.internal.jdbc.JdbcUtils.openConnection(JdbcUtils.java:60)
	at org.flywaydb.core.internal.jdbc.JdbcConnectionFactory.<init>(JdbcConnectionFactory.java:75)
	at org.flywaydb.core.FlywayExecutor.execute(FlywayExecutor.java:146)
	at org.flywaydb.core.Flyway.clean(Flyway.java:213)
	at scala.meta.internal.metals.Tables.migrateOrRestart(Tables.scala:156)
	at scala.meta.internal.metals.Tables.tryUrl(Tables.scala:144)
	at scala.meta.internal.metals.Tables.persistentConnection(Tables.scala:138)
	at scala.meta.internal.metals.Tables.tryNoAutoServer(Tables.scala:98)
	at scala.meta.internal.metals.Tables.tryAutoServer(Tables.scala:92)
	at scala.meta.internal.metals.Tables.connect(Tables.scala:48)
	at scala.meta.internal.metals.Tables.connection(Tables.scala:74)
	at scala.meta.internal.metals.Tables.$anonfun$fingerprints$1(Tables.scala:39)
	at scala.meta.internal.metals.Fingerprints.load(Fingerprints.scala:52)
	at scala.meta.internal.metals.MetalsLanguageServer.$anonfun$initialize$2(MetalsLanguageServer.scala:882)
	at scala.concurrent.Future$.$anonfun$apply$1(Future.scala:678)
	at scala.concurrent.impl.Promise$Transformation.run(Promise.scala:467)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
	at java.lang.Thread.run(Thread.java:833)
Caused by: org.h2.jdbc.JdbcSQLNonTransientException: IO Exception: "/sgoinfre/scristia/codam/cub3d/.metals/metals.mv.db" [90028-214]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:554)
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:477)
	at org.h2.message.DbException.get(DbException.java:212)
	at org.h2.mvstore.db.Store.convertMVStoreException(Store.java:171)
	at org.h2.mvstore.db.Store.<init>(Store.java:145)
	at org.h2.engine.Database.<init>(Database.java:324)
	at org.h2.engine.Engine.openSession(Engine.java:92)
	at org.h2.engine.Engine.openSession(Engine.java:222)
	at org.h2.engine.Engine.createSession(Engine.java:201)
	at org.h2.engine.SessionRemote.connectEmbeddedOrServer(SessionRemote.java:338)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:122)
	at org.h2.Driver.connect(Driver.java:59)
	at org.flywaydb.core.internal.jdbc.DriverDataSource.getConnectionFromDriver(DriverDataSource.java:258)
	at org.flywaydb.core.internal.jdbc.DriverDataSource.getConnection(DriverDataSource.java:223)
	at org.flywaydb.core.internal.jdbc.JdbcUtils.openConnection(JdbcUtils.java:48)
	at org.flywaydb.core.internal.jdbc.JdbcConnectionFactory.<init>(JdbcConnectionFactory.java:75)
	at org.flywaydb.core.FlywayExecutor.execute(FlywayExecutor.java:146)
	at org.flywaydb.core.Flyway.clean(Flyway.java:213)
	at scala.meta.internal.metals.Tables.migrateOrRestart(Tables.scala:156)
	at scala.meta.internal.metals.Tables.tryUrl(Tables.scala:144)
	at scala.meta.internal.metals.Tables.persistentConnection(Tables.scala:138)
	at scala.meta.internal.metals.Tables.tryNoAutoServer(Tables.scala:98)
	at scala.meta.internal.metals.Tables.tryAutoServer(Tables.scala:92)
	at scala.meta.internal.metals.Tables.connect(Tables.scala:48)
	at scala.meta.internal.metals.Tables.connection(Tables.scala:74)
	at scala.meta.internal.metals.Tables.$anonfun$fingerprints$1(Tables.scala:39)
	at scala.meta.internal.metals.Fingerprints.load(Fingerprints.scala:52)
	at scala.meta.internal.metals.MetalsLanguageServer.$anonfun$initialize$2(MetalsLanguageServer.scala:882)
	at scala.concurrent.Future$.$anonfun$apply$1(Future.scala:678)
	at scala.concurrent.impl.Promise$Transformation.run(Promise.scala:467)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
	at java.lang.Thread.run(Thread.java:833)
Caused by: org.h2.mvstore.MVStoreException: Could not open file /sgoinfre/scristia/codam/cub3d/.metals/metals.mv.db [2.1.214/1]
	at org.h2.mvstore.DataUtils.newMVStoreException(DataUtils.java:1004)
	at org.h2.mvstore.FileStore.open(FileStore.java:185)
	at org.h2.mvstore.FileStore.open(FileStore.java:128)
	at org.h2.mvstore.MVStore.<init>(MVStore.java:452)
	at org.h2.mvstore.MVStore$Builder.open(MVStore.java:4082)
	at org.h2.mvstore.db.Store.<init>(Store.java:136)
	at org.h2.engine.Database.<init>(Database.java:324)
	at org.h2.engine.Engine.openSession(Engine.java:92)
	at org.h2.engine.Engine.openSession(Engine.java:222)
	at org.h2.engine.Engine.createSession(Engine.java:201)
	at org.h2.engine.SessionRemote.connectEmbeddedOrServer(SessionRemote.java:338)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:122)
	at org.h2.Driver.connect(Driver.java:59)
	at org.flywaydb.core.internal.jdbc.DriverDataSource.getConnectionFromDriver(DriverDataSource.java:258)
	at org.flywaydb.core.internal.jdbc.DriverDataSource.getConnection(DriverDataSource.java:223)
	at org.flywaydb.core.internal.jdbc.JdbcUtils.openConnection(JdbcUtils.java:48)
	at org.flywaydb.core.internal.jdbc.JdbcConnectionFactory.<init>(JdbcConnectionFactory.java:75)
	at org.flywaydb.core.FlywayExecutor.execute(FlywayExecutor.java:146)
	at org.flywaydb.core.Flyway.clean(Flyway.java:213)
	at scala.meta.internal.metals.Tables.migrateOrRestart(Tables.scala:156)
	at scala.meta.internal.metals.Tables.tryUrl(Tables.scala:144)
	at scala.meta.internal.metals.Tables.persistentConnection(Tables.scala:138)
	at scala.meta.internal.metals.Tables.tryNoAutoServer(Tables.scala:98)
	at scala.meta.internal.metals.Tables.tryAutoServer(Tables.scala:92)
	at scala.meta.internal.metals.Tables.connect(Tables.scala:48)
	at scala.meta.internal.metals.Tables.connection(Tables.scala:74)
	at scala.meta.internal.metals.Tables.$anonfun$fingerprints$1(Tables.scala:39)
	at scala.meta.internal.metals.Fingerprints.load(Fingerprints.scala:52)
	at scala.meta.internal.metals.MetalsLanguageServer.$anonfun$initialize$2(MetalsLanguageServer.scala:882)
	at scala.concurrent.Future$.$anonfun$apply$1(Future.scala:678)
	at scala.concurrent.impl.Promise$Transformation.run(Promise.scala:467)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
	at java.lang.Thread.run(Thread.java:833)
Caused by: java.io.IOException: Operation not supported
	at sun.nio.ch.FileDispatcherImpl.lock0(Native Method)
	at sun.nio.ch.FileDispatcherImpl.lock(FileDispatcherImpl.java:96)
	at sun.nio.ch.FileChannelImpl.tryLock(FileChannelImpl.java:1304)
	at java.nio.channels.FileChannel.tryLock(FileChannel.java:1191)
	at org.h2.mvstore.FileStore.open(FileStore.java:169)
	at org.h2.mvstore.FileStore.open(FileStore.java:128)
	at org.h2.mvstore.MVStore.<init>(MVStore.java:452)
	at org.h2.mvstore.MVStore$Builder.open(MVStore.java:4082)
	at org.h2.mvstore.db.Store.<init>(Store.java:136)
	at org.h2.engine.Database.<init>(Database.java:324)
	at org.h2.engine.Engine.openSession(Engine.java:92)
	at org.h2.engine.Engine.openSession(Engine.java:222)
	at org.h2.engine.Engine.createSession(Engine.java:201)
	at org.h2.engine.SessionRemote.connectEmbeddedOrServer(SessionRemote.java:338)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:122)
	at org.h2.Driver.connect(Driver.java:59)
	at org.flywaydb.core.internal.jdbc.DriverDataSource.getConnectionFromDriver(DriverDataSource.java:258)
	at org.flywaydb.core.internal.jdbc.DriverDataSource.getConnection(DriverDataSource.java:223)
	at org.flywaydb.core.internal.jdbc.JdbcUtils.openConnection(JdbcUtils.java:48)
	at org.flywaydb.core.internal.jdbc.JdbcConnectionFactory.<init>(JdbcConnectionFactory.java:75)
	at org.flywaydb.core.FlywayExecutor.execute(FlywayExecutor.java:146)
	at org.flywaydb.core.Flyway.clean(Flyway.java:213)
	at scala.meta.internal.metals.Tables.migrateOrRestart(Tables.scala:156)
	at scala.meta.internal.metals.Tables.tryUrl(Tables.scala:144)
	at scala.meta.internal.metals.Tables.persistentConnection(Tables.scala:138)
	at scala.meta.internal.metals.Tables.tryNoAutoServer(Tables.scala:98)
	at scala.meta.internal.metals.Tables.tryAutoServer(Tables.scala:92)
	at scala.meta.internal.metals.Tables.connect(Tables.scala:48)
	at scala.meta.internal.metals.Tables.connection(Tables.scala:74)
	at scala.meta.internal.metals.Tables.$anonfun$fingerprints$1(Tables.scala:39)
	at scala.meta.internal.metals.Fingerprints.load(Fingerprints.scala:52)
	at scala.meta.internal.metals.MetalsLanguageServer.$anonfun$initialize$2(MetalsLanguageServer.scala:882)
	at scala.concurrent.Future$.$anonfun$apply$1(Future.scala:678)
	at scala.concurrent.impl.Promise$Transformation.run(Promise.scala:467)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
	at java.lang.Thread.run(Thread.java:833)

2023.01.13 14:58:05 INFO  time: initialize in 3.27s
2023.01.13 14:58:07 WARN  Build server is not auto-connectable.
2023.01.13 14:58:07 WARN  no build tool detected in workspace '/sgoinfre/scristia/codam/cub3d'. The most common cause for this problem is that the editor was opened in the wrong working directory, for example if you use sbt then the workspace directory should contain build.sbt. 
Jan 13, 2023 2:58:16 PM org.eclipse.lsp4j.jsonrpc.services.GenericEndpoint notify
INFO: Unsupported notification method: $/setTrace
Jan 13, 2023 2:58:23 PM org.eclipse.lsp4j.jsonrpc.services.GenericEndpoint notify
INFO: Unsupported notification method: $/setTrace
Jan 13, 2023 2:58:24 PM org.eclipse.lsp4j.jsonrpc.services.GenericEndpoint notify
INFO: Unsupported notification method: $/setTrace
Jan 13, 2023 2:58:31 PM org.eclipse.lsp4j.jsonrpc.services.GenericEndpoint notify
INFO: Unsupported notification method: $/setTrace
Jan 13, 2023 2:58:35 PM org.eclipse.lsp4j.jsonrpc.services.GenericEndpoint notify
INFO: Unsupported notification method: $/setTrace
Jan 13, 2023 2:58:37 PM org.eclipse.lsp4j.jsonrpc.services.GenericEndpoint notify
INFO: Unsupported notification method: $/setTrace
Jan 13, 2023 2:58:51 PM org.eclipse.lsp4j.jsonrpc.services.GenericEndpoint notify
INFO: Unsupported notification method: $/setTrace
Jan 13, 2023 2:58:57 PM org.eclipse.lsp4j.jsonrpc.services.GenericEndpoint notify
INFO: Unsupported notification method: $/setTrace
Jan 13, 2023 2:59:16 PM org.eclipse.lsp4j.jsonrpc.services.GenericEndpoint notify
INFO: Unsupported notification method: $/setTrace
Jan 13, 2023 2:59:23 PM org.eclipse.lsp4j.jsonrpc.services.GenericEndpoint notify
INFO: Unsupported notification method: $/setTrace
Jan 13, 2023 2:59:29 PM org.eclipse.lsp4j.jsonrpc.services.GenericEndpoint notify
INFO: Unsupported notification method: $/setTrace
Jan 13, 2023 2:59:34 PM org.eclipse.lsp4j.jsonrpc.services.GenericEndpoint notify
INFO: Unsupported notification method: $/setTrace
Jan 13, 2023 2:59:38 PM org.eclipse.lsp4j.jsonrpc.services.GenericEndpoint notify
INFO: Unsupported notification method: $/setTrace
Jan 13, 2023 2:59:48 PM org.eclipse.lsp4j.jsonrpc.services.GenericEndpoint notify
INFO: Unsupported notification method: $/setTrace
Jan 13, 2023 3:00:26 PM org.eclipse.lsp4j.jsonrpc.services.GenericEndpoint notify
INFO: Unsupported notification method: $/setTrace
Jan 13, 2023 3:00:28 PM org.eclipse.lsp4j.jsonrpc.services.GenericEndpoint notify
INFO: Unsupported notification method: $/setTrace
Jan 13, 2023 3:01:09 PM org.eclipse.lsp4j.jsonrpc.services.GenericEndpoint notify
INFO: Unsupported notification method: $/setTrace
Jan 13, 2023 3:01:10 PM org.eclipse.lsp4j.jsonrpc.services.GenericEndpoint notify
INFO: Unsupported notification method: $/setTrace
Jan 13, 2023 3:01:37 PM org.eclipse.lsp4j.jsonrpc.services.GenericEndpoint notify
INFO: Unsupported notification method: $/setTrace
